src/test/java/com/spotify/folsom/transcoder/SerializableObjectTranscoderTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2015 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.spotify.folsom.transcoder;
+
+import org.junit.Test;
+
+import java.io.Serializable;
+
+import static org.junit.Assert.assertEquals;
+
+public class SerializableObjectTranscoderTest {
+
+  private static final class SerializableTestObject implements Serializable {
+    private String value1 = "hello";
+    private String value2 = "world";
+    private int value3 = 5;
+  }
+
+  @Test
+  public void testEncodeDecode() {
+    SerializableTestObject b = new SerializableTestObject();
+
+    final byte[] encoded = SerializableObjectTranscoder.INSTANCE.encode(b);
+    final SerializableTestObject testObject =
+            (SerializableTestObject) SerializableObjectTranscoder.INSTANCE.decode(encoded);
+
+    assertEquals("hello", testObject.value1);
```

Comment:
I don't trust this test since the asserts are using the same values as the class defaults - maybe remove the explicit defaults and set the fields manually before serializing?